src/elastio-snap.c

```diff
@@ -3344,6 +3346,15 @@ static void __on_bio_read_complete(struct bio *bio, int err){
 	bio_free_clone(bio);
 }
 
+static inline void wait_for_bio_complete(struct snap_device *dev)
+{
+	struct bio_queue *bq = &dev->sd_cow_bios;
```

Comment:
I'd like to see a comment here linking to the Github issue this is resolving.

Reply:
@anelson I don't think any comments are necessary: this PR implement a missing piece of functionality, which should be here despite any issues found.

Reply:
I disagree.  This was missing for years and we noticed it only in a minor test failure on one distro.  It's clearly not obvious.

Reply:
@anelson done